src/libratbag-data.c

```diff
@@ -272,6 +280,13 @@ ratbag_device_data_get_name(const struct ratbag_device_data *data)
 	return data->name;
 }
 
+const char *
+ratbag_device_data_get_device_type(const struct ratbag_device_data *data)
+{
+	return data->device_type;
```

Comment:
if we'd need this (see my comment about enums on the dbus interface though) we can just do this as

```c
if (type == TYPE_MOUSE) return "mouse";
```